cmd/frcli/node_audit.go

```diff
@@ -194,6 +194,13 @@ func queryOnChainReport(ctx *cli.Context) error {
 		}
 	}()
 
+	if len(report.Reports) > 0 {
+		CSVHeaders = strings.Replace(
```

Comment:
Instead of `strings.Replace` we could have: 

```
var CSVHeaders = "Timestamp,OnChain,Type,Category,Amount(Msat),Amount(%v),TxID,Reference,BTCPrice,BTCTimestamp,Note"

headers = fmt.Sprintf(CSVHeaders, report.Reports[0].BtcPrice.Currency)
```

Just a little shorter, and gives us options to add more variable fields going forward.